frame/collective/src/lib.rs

```diff
@@ -71,6 +71,52 @@ pub type ProposalIndex = u32;
 /// vote exactly once, therefore also the number of votes for any given motion.
 pub type MemberCount = u32;
 
+/// Default voting strategy when a member is inactive.
+pub trait DefaultVote {
+	/// Get the default voting strategy, given:
+	///
+	/// - Whether the prime member voted Aye.
+	/// - Raw number of yes votes.
+	/// - Raw number of no votes.
+	/// - Total number of member count.
+	fn default_vote(
+		prime_voted_aye: bool,
```

Comment:
why not providing the complete prime_vote information as `Option<bool>` ?

and then rename PrimeDefaultVote to PrimeOrFalseDefaultVote ?

Reply:
Changed it to expose the whole `prime_vote`! A small grumble on the rename though -- I think `PrimeDefaultVote` already expresses the whole intention, and `PrimeOrFalseDefaultVote` is slightly redundant. If we rename it, we also need to rename `MoreThanMajorityThenPrimeDefaultVote` to `MoreThanMajorityThenPrimeOrFalseDefaultVote` which I think is too long.

Reply:
Yes I agree